dbdump/run.sh.tpl

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+set -e
+
+log_info() {
+    time=$(date +"%Y-%m-%dT%H:%M:%SZ")
+    echo "{\"level\":\"info\",\"time\":\"$time\",\"message\":\"$1\"}"
+}
+
+log_fatal() {
+    time=$(date +"%Y-%m-%dT%H:%M:%SZ")
+    echo "{\"level\":\"fatal\",\"time\":\"$time\",\"message\":\"$1\"}"
+}
+
+log_warn() {
+    time=$(date +"%Y-%m-%dT%H:%M:%SZ")
+    echo "{\"level\":\"warn\",\"time\":\"$time\",\"message\":\"$1\"}"
+}
+
+log_info "run serlo.org dbdump"
+
+connect="--host=${database_host} --port=${database_port} --user=${database_username} --password=${database_password}"
+
+set +e
+mysql $connect -e "SHOW DATABASES; USE ${database_name}; SHOW TABLES;" | grep uuid >/dev/null 2>/dev/null
+if [[ $? != 0 ]] ; then
+    log_info "database ${database_name} does not exist; nothing to dump"
+    exit 0
+fi
+set -e
+
+cd /tmp
+
+log_info "dump serlo.org database - start"
+log_info "dump database schema"
+
+mysqldump $connect --no-data --lock-tables=false --add-drop-database serlo >dump.sql
+
+log_info "dump database data"
+mysqldump $connect --no-create-info --lock-tables=false --add-locks --ignore-table=serlo.session serlo >> dump.sql
+
+log_info "anonymize database dump"
+sed -i -r "/([0-9]+, ?)'[^']+\@[^']+', ?'[^']+', ?'[^']+',( ?[0-9]+, ?'[^']+', ?[0-9], ?)'[^']+'/ s//\1CONCAT\(LEFT\(UUID\(\), 8\),'@localhost'\), LEFT\(UUID\(\), 8\), '8a534960a8a4c8e348150a0ae3c7f4b857bfead4f02c8cbf0d',\2LEFT\(UUID\(\), 8\)/" dump.sql
+
```

Comment:
This is probably outdated because of the database changes in https://github.com/serlo/serlo.org/pull/77, unless you changed something here? I think I have the updated script somewhere...

Reply:
See Slack. Feel free to push the current script. The code here is the most recent one I had